src/index.html

```diff
@@ -1,12 +1,372 @@
 <!doctype html>
-<html lang="en">
+<html class="catalog-mac" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items"> 
+            <a href="#" class="nav__link is-active" >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              samsung
+            </a> 
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              smartphones
+            </a> 
+          </li>
+            
+          <li class="nav__items"> 
+            <a 
+              class="nav__link" 
+              href="#" 
+              data-qa="nav-hover"
+              >
+              laptops & computers
+            </a>
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              gadgets
+            </a> 
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              tablets
+            </a>
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              photo
+            </a> 
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="products">
+        <div class="card" data-qa="card">
+          <div class="card__picture">
+            <img 
+              class="card__images" 
+              src="images/imac.jpeg" 
+              alt="imac"
+            >
+          </div>
+          <h2 class="card__name">
```

Comment:
- Fix it everywhere 
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/99403198/213871825-7f1b6256-13d7-4598-a959-9dbe6fc73b38.png">

```suggestion
          </div>
          
          <h2 class="card__name">
```